### PR TITLE
do not exit linutil in confirmation prompt when q is pressed

### PR DESCRIPTION
**Pull request by nnyyxxxx**
## Type of Change
- [x] Bug fix

https://github.com/user-attachments/assets/e7ee90e5-fa73-4823-a284-597c361d2f25

### CODE REVIEW

File: tui/src/confirmation.rs [hunk 1]
```diff
@@ -88,7 +88,7 @@ impl FloatContent for ConfirmPrompt {
         use KeyCode::*;
         self.status = match key.code {
             Char('y') | Char('Y') => ConfirmStatus::Confirm,
-            Char('n') | Char('N') | Esc => ConfirmStatus::Abort,
+            Char('n') | Char('N') | Esc | Char('q') | Char('Q') => ConfirmStatus::Abort,
```
Comment (adamperkowski) [inline tui/src/confirmation.rs:91]: We have to be consistent with this. Here you're doing `Char('q') | Char('Q')` but other modules like `float.rs` or `state.rs` use `Char('q')` only.

File: tui/src/state.rs [hunk 2]
```diff
@@ -411,11 +411,9 @@ impl AppState {
...
-            self.focus,
-            Focus::TabList | Focus::List | Focus::ConfirmationPrompt(_)
-        ) && (key.code == KeyCode::Char('q')
-            || key.modifiers.contains(KeyModifiers::CONTROL) && key.code == KeyCode::Char('c'))
+        if matches!(self.focus, Focus::TabList | Focus::List)
+            && (key.code == KeyCode::Char('q')
+                || key.modifiers.contains(KeyModifiers::CONTROL) && key.code == KeyCode::Char('c'))
         {
             return false;
         }
```
Comment (adamperkowski) [inline tui/src/state.rs:419]: you forgot about `Ctrl-C`

```suggestion
        }
        
        if matches!(self.focus, Focus::ConfirmationPrompt(_))
            && (key.modifiers.contains(KeyModifiers::CONTROL) && key.code == KeyCode::Char('c'))
        {
            return false;
        }
```

**Review by adamperkowski — CHANGES_REQUESTED** [review]

**Comment by adamperkowski** [comment]
cargo fmt seeing ghosts lol. wants to change `{` to `{`

**Comment by adamperkowski** [comment]
9641e6474c6e73581f9ce9f846e36f9174b56808 Excuse me what do you mean untested? It literally does what it says.
image
It's not supposed to close the popup on ctrl-c, it's supposed to exit.

**Comment by nnyyxxxx** [comment]
> [9641e64](https://github.com/ChrisTitusTech/linutil/commit/9641e6474c6e73581f9ce9f846e36f9174b56808) Excuse me what do you mean untested? It literally does what it says. image It's not supposed to close the popup on ctrl-c, it's supposed to exit.

ah

File: tui/src/state.rs [hunk 3]
```diff
@@ -411,15 +411,23 @@ impl AppState {
...
 
+        if matches!(self.focus, Focus::ConfirmationPrompt(_))
+            && (key.modifiers.contains(KeyModifiers::CONTROL) && key.code == KeyCode::Char('c'))
+        {
+            self.focus = Focus::List;
+            if !self.multi_select {
+                self.selected_commands.clear();
+            }
+            return true;
```
Comment (adamperkowski) [inline tui/src/state.rs:428]: image

No need for `Ctrl-C` to do the same thing. 

```suggestion
            return false;
```

**Review by adamperkowski — CHANGES_REQUESTED** [review]

**Review by adamperkowski — APPROVED** [review]
🍟

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**